packages/gatsby/src/redux/types.ts

```diff
@@ -105,7 +109,7 @@ export interface IGatsbyConfig {
   jsxRuntime?: "classic" | "automatic"
   jsxImportSource?: string
   trailingSlash?: TrailingSlash
-  graphqlTypegen?: boolean
+  graphqlTypegen?: boolean | IGraphQLTypegenOptions
```

Comment:
Should this be just `IGraphQLTypegenOptions` here? or alternatively just `false | IGraphQLTypegenOptions`? With `.custom` converting `true` to full blown object, we shouldn't ever get actual `true` value here anymore?